endoreg_db_api/settings_prod/__init__.py

```
@@ -17,12 +17,17 @@
     DEFAULT_AUTO_FIELD,
     STATIC_ROOT,
     STATICFILES_DIRS,
+    CORS_ALLOWED_ORIGINS,  # by gc-08,need to delete after testing/correcting them
```

Ruff (0.11.9)

`endoreg_db_api.settings_base.CORS_ALLOWED_ORIGINS` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)